go.mod

```diff
@@ -9,6 +9,7 @@ require (
 	github.com/golang-jwt/jwt/v4 v4.5.0
 	github.com/google/go-containerregistry v0.20.2
 	github.com/munnerz/goautoneg v0.0.0-20191010083416-a7dc8b61c822
+	github.com/olekukonko/tablewriter v0.0.5
```

Comment:
Do we have a concern that this is not really maintained? https://github.com/olekukonko/tablewriter does not have an update for more than a year now.

Reply:
Yeah, that's a valid concern. I followed up on an issue they have where the maintenance of the project was already put into question. It looks like it suffers from what other open-source librabries suffer as well, they slowly start to become abandoned. It looks like https://github.com/jedib0t/go-pretty could be a suitable alternative. I will look into it.